ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/helpers/RequestHelper.java

```diff
@@ -117,4 +138,35 @@ public static String generateParameterCheckingBlock(final Request request) {
 
         return builder.toString();
     }
+
+    public static String generateInitParamBlock(final Parameter parm, final boolean isRequired) {
```

Comment:
We should think of a way to refactor this method to make it cleaner and easier to understand.